session/bootstrap.go

```diff
@@ -728,6 +728,9 @@ const (
 	// The variable name in mysql.tidb table and it records the default value of
 	// oom-action when upgrade from v3.0.x to v4.0.11+.
 	tidbDefOOMAction = "default_oom_action"
+
+	tiDBStatsGCLastTS        = "tidb_stats_gc_last_ts"
```

Comment:
We defined this const string twice (another in `statistics/handle/gc.go`), we can merge them to make it more clear when other people are trying to find their usages.